src/index.tsx

```diff
@@ -133,7 +135,7 @@ export function configureAuth<
       return renderLoading();
     }
 
-    return <div>Unhandled status: {status}</div>;
+    return renderError(status);
```

Comment:
Here we can render the error:

To do that, we can replace the following:

```tsx
return renderError(status);
```
With the following:
```ts
if (status === 'error') {
  return renderError(error);
}

return null;
```